test/sites/bigFont.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="pl">
+<head>
+    <meta charset="UTF-8">
+    <style>
+        #fontWrapper p {margin:0 5px 0 0; float:left; clear:both;}
+        #fontWrapper ul {list-style-type:none; float:left; padding:0; margin:0}
+        #fontWrapper li {float:left;  margin:0 3px 0 0;}
+        #fontWrapper li a {display:block; width:24px; height:15px; text-indent:-9999em; border:none; outline:none;}
+        #fontWrapper li a.fontDefault {background:url(http://wcag20.widzialni.org/layout/images/fontDefault.png) no-repeat; width:20px;}
+        #fontWrapper li a.fontBigger {background:url(http://wcag20.widzialni.org/layout/images/fontBigger.png) no-repeat}
+        #fontWrapper li a.fontBig {background:url(http://wcag20.widzialni.org/layout/images/fontBig.png) no-repeat}
+
+        #fontWrapper li a.fontDefault:focus,
+        #fontWrapper li a.fontBigger:focus,
+        #fontWrapper li a.fontBig:focus { border:1px solid #000; }
+    </style>
+</head>
+<body>
+
+<div id="fontWrapper">
+    <p>Czcionka:</p>
+    <ul>
+        <li><a href="X" class="fontDefault" title="Czcionka domyślna">Czcionka domyślna</a></li>
+        <li><a href="X" class="fontBigger" title="Czcionka średnia">Czcionka średnia</a></li>
+        <li><a href="X" class="fontBig" title="Czcionka duża">Czcionka duża</a></li>
+    </ul>
+</div>
+
+</body>
+</html>
```

Comment:
Wiem, że słabe ale desperacko próbuję powiększać liczbę metryk ...
Ale z drugiej strony bardzo słabe nie jest, bo schemat jest powtarzający
się i często kopiowany.
Plan jest taki aby puścić metrykę na całej bazie wyłapać te strony gdzie
jest 0, a potem manualnie sprawdzić czy rzeczywiście niema i jak jest to
spróbować to podrasować.


2015-09-05 23:51 GMT+02:00 Bartosz Gałek <notifications@github.com>:

> In test/sites/bigFont.html
> <https://github.com/pwd-project/pwd-analysis/pull/9#discussion_r38815122>:
>
> > +        #fontWrapper li a.fontBig:focus { border:1px solid #000; }
> > +    </style>
> > +</head>
> > +<body>
> > +
> > +<div id="fontWrapper">
> > +    <p>Czcionka:</p>
> > +    <ul>
> > +        <li><a href="X" class="fontDefault" title="Czcionka domyślna">Czcionka domyślna</a></li>
> > +        <li><a href="X" class="fontBigger" title="Czcionka średnia">Czcionka średnia</a></li>
> > +        <li><a href="X" class="fontBig" title="Czcionka duża">Czcionka duża</a></li>
> > +    </ul>
> > +</div>
> > +
> > +</body>
> > +</html>
>
> nawet github mowi, no newline
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pwd-project/pwd-analysis/pull/9/files#r38815122>.
>